src/panels/calendar/dialog-calendar-event-detail.ts

```diff
@@ -125,7 +127,9 @@ class DialogCalendarEventDetail extends LitElement {
     // TODO: Make sure this handles translations
     try {
       const readableText =
-        value === "" ? "" : RRule.fromString(`RRULE:${value}`).toText();
+        value === ""
+          ? ""
+          : capitalizeFirstLetter(RRule.fromString(`RRULE:${value}`).toText());
```

Comment:
Forgot to mention that, but is by design to produce a cleaner look. E.g. the lowest element in the currently showing detail (the last-updated info also starts with a capital letter, although in EN that is not really visible since there a number is at the front usually) and the calendar name probably also almost is starting a capital, although that one is up to the user.

Reply:
Ah, makes sense, good catch.